backend/src/main/java/fk/prof/backend/BackendManager.java

```diff
@@ -163,6 +165,13 @@ private void initStorage() {
     this.bufferPool = new GenericObjectPool<>(new ByteBufferPoolFactory(bufferPoolConfig.getInteger("buffer.size"), false), poolConfig);
   }
 
+  private PolicyStore createPolicyStore(CuratorFramework curatorClient) {
+    JsonObject policyConfig = configManager.getPolicyConfig();
+    String policyPath = policyConfig.getString("policy.path", "/policy");
```

Comment:
Let's add a method in ConfigManager to get policy path, instead of above two lines